topic_name : /scan
queue_size : 10
p_gain : 10.0